Dockerfile

```diff
@@ -0,0 +1,41 @@
+
+FROM ubuntu:18.04
+RUN apt-get update
+RUN apt-get install -y sudo build-essential git axel wget curl
+
+
+# for python 3.10
+RUN sudo apt install software-properties-common -y
+RUN sudo add-apt-repository ppa:deadsnakes/ppa
+RUN DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get -y install tzdata
+RUN sudo apt-get -y install python3.10
+RUN apt-get -y install python3-numpy python3-scipy python3-pip
+
+# Get Rust; NOTE: using sh for better compatibility with other base images
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
+
+# Add .cargo/bin to PATH
+ENV PATH="/root/.cargo/bin:${PATH}"
+
+# Check cargo is visible
```

Comment:
Did you mean to add a command to do this?

Reply:
no. will remove comment.